tb2_controller:
  ros__parameters:
    timer_period: 0.1
    path_name: "eight"
    path_execution: True